bin/makefile-openbsd.x86_64-x

```diff
@@ -0,0 +1,54 @@
+# Options for OpenBSD, Intel x86_64 and X-Window
+
+#CC = gcc -m64 $(GCC_CFLAGS) -fno-omit-frame-pointer -Wall -Wextra -fno-aggressive-loop-optimizations
```

Comment:
The flags after the `GCC_FLAGS` can be removed. I have an open PR removing them from other files.